.github/workflows/haskell.yml

```
@@ -645,3 +645,24 @@ jobs:
           echo 'The diff must not contain any `undefined` values'
           false
         fi
+
+  changelog-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install changelog linter
+      run: |
+        VERSION=0.1.0.0
+        BINDIR=$HOME/.local/bin
+        mkdir -p "$BINDIR"
+        curl -sSfL "https://github.com/neilmayhew/changelogs/releases/download/$VERSION/changelogs-Linux-x86_64.zip" -o changelogs.zip
```

Could you please change this to an IOHK repo.
I'd prefer not to depend on tools in our personal accounts:
```suggestion
        curl -sSfL "https://github.com/input-output-hk/cardano-ledger-release-tool/releases/download/$VERSION/changelogs-Linux-x86_64.zip" -o changelogs.zip
```

---

I will move it, but it's taking longer than I expected. So, to get the cleanups to the changelogs merged, I've taken that commit out of this PR. When this is merged, I'll rebase my original PR and rework the commit that adds the linting to CI.

---

I've now finished moving the linter to the new repo, and I've re-added the commit that adds it to CI.

Since you requested changes, you need to re-approve this PR before it can be merged.